client/src/app/site/pages/meetings/modules/participant-search-selector/components/participant-search-selector/participant-search-selector.component.ts

```diff
@@ -161,9 +161,9 @@ export class ParticipantSearchSelectorComponent extends BaseUiComponent implemen
         );
         this.snackBar.open(
             this.translate
-                .instant(`A user with the username 'USERNAME' and the surname 'LAST_NAME' was created.`)
+                .instant(`A user with the username 'USERNAME' and the first name 'FIRST_NAME' was created.`)
```

Comment:
AFAIK we currently use two different replacement systems for translations:
- Simply using `{}` and replacing by order, for example here: https://github.com/OpenSlides/openslides-client/blob/main/client/src/app/site/modules/translations/marked-translations.ts#L51
- encasing the variable in percent signs, I think only used when sending emails: https://github.com/OpenSlides/openslides-client/blob/main/client/src/app/site/services/user-controller.service.ts#L209

Unless @luisa-beerboom or @bastianjoel know that you method is used anywhere, I would argue that we do not want a third way of translating variables and you should change it to one of the above.

Reply:
I changed the var names to %username% and %first_name%. It is just a renaming.